Tests/JWTTests/JWKTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+@testable import JWT
+
+class JWKTests: XCTestCase {
+
+    static let allTests = [
```

Comment:
Instead of supplying an `allTests` array, just run `swift test --generate-linuxmain` before committing.